### PR TITLE
Refactor test(s) that check for parameter types to use assert.throws() call #153

### PR DESCRIPTION
**Pull request by erangeles**

### CODE REVIEW

File: tests/unit/services/sl-translate-test.js [hunk 1]
```diff
@@ -28,139 +28,67 @@ test( 'dictionary property defaults to null', function( assert ) {
...
-    try {
-        TS.setDictionary( {} );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        !assertionThrown,
-        'Parameter was an Object'
+    assert.equal(
```
Comment (erangeles) [inline tests/unit/services/sl-translate-test.js]: The setDictionary function returns undefined, therefore we cannot use assert.ok, that is the reason assert.equal tests are included.

File: tests/unit/services/sl-translate-test.js [hunk 2]
```diff
@@ -28,139 +28,67 @@ test( 'dictionary property defaults to null', function( assert ) {
...
 
-    assert.ok(
-        assertionThrown,
-        'Parameter was empty'
+    assert.throws(
+        callSetDictionary,
+        'Parameter was an array'
     );
 
-    // Null parameter
```
Comment (notmessenger) [inline tests/unit/services/sl-translate-test.js]: Leave these comments in before each new type check

File: tests/unit/services/sl-translate-test.js [hunk 3]
```diff
@@ -28,139 +28,67 @@ test( 'dictionary property defaults to null', function( assert ) {
 });
 
 test( 'setDictionary() accepts only an object as a parameter', function( assert ) {
```
Comment (notmessenger) [inline tests/unit/services/sl-translate-test.js]: `undefined` is not being tested

**Comment by erangeles** [comment]
Undefined test added, param comments added as well.

**Pull request merged**